pkg/agent/cniserver/server.go

```diff
@@ -106,10 +107,10 @@ type CNIServer struct {
 	kubeClient           clientset.Interface
 	containerAccess      *containerAccessArbitrator
 	podConfigurator      *podConfigurator
-	// podUpdates is a channel for notifying Pod updates to other components, i.e NetworkPolicyController.
-	podUpdates  chan<- v1beta2.PodReference
-	isChaining  bool
-	routeClient route.Interface
+	// entityUpdates is a channel for notifying Pod updates to other components, i.e NetworkPolicyController.
```

Comment:
suggestion: entityUpdates is a channel for notifying updates of local endpoints / entities (most notably Pod) to other components which may benefit from this information, i.e NetworkPolicyController.

Reply:
Done, thanks for the suggestions